v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -49,6 +49,26 @@ export const ERROR_CATEGORIES: {
 } = {
   GENERAL: { min: 1, max: 99, websiteTitle: "General errors" },
   INTERNAL: { min: 100, max: 199, websiteTitle: "Internal Hardhat errors" },
+  TASK_DEFINITIONS: {
```

Comment:
@alcuadrado can I double check.
Is the intention that we try and keep the same error codes across v2 and v3?
Or do we consider ourselves now free to define new error codes and categories?

Reply:
No, not necessary. We'll format them differently so that there are no clashes.